parser/parser.y

```diff
@@ -1598,36 +1598,28 @@ ResourceGroupOptionList:
 	{
 		$$ = []*ast.ResourceGroupOption{$1.(*ast.ResourceGroupOption)}
 	}
-|	ResourceGroupOptionList DirectResourceGroupOption
+|	DirectResourceGroupOption "BURSTABLE"
 	{
-		$$ = append($1.([]*ast.ResourceGroupOption), $2.(*ast.ResourceGroupOption))
+		$$ = append([]*ast.ResourceGroupOption{$1.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
 	}
-|	ResourceGroupOptionList ',' DirectResourceGroupOption
+|	DirectResourceGroupOption ',' "BURSTABLE"
 	{
-		$$ = append($1.([]*ast.ResourceGroupOption), $3.(*ast.ResourceGroupOption))
+		$$ = append([]*ast.ResourceGroupOption{$1.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
 	}
+|	"BURSTABLE" DirectResourceGroupOption
+    {
+    	$$ = append([]*ast.ResourceGroupOption{$2.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
+    }
+|	"BURSTABLE" ',' DirectResourceGroupOption
+    {
+    	$$ = append([]*ast.ResourceGroupOption{$3.(*ast.ResourceGroupOption)}, &ast.ResourceGroupOption{Tp: ast.ResourceBurstableOpiton, BoolValue: true})
+    }
```

Comment:
It looks confusing... **where BURSTABLE is allowed exactly**?


```
create resource group xx  BURSTABLE RU_PER_SEC = 123 BURSTABLE , BURSTABLE RU_PER_SEC  = 456
```

Reply:
I reworked the yacc file to make logic more readable.